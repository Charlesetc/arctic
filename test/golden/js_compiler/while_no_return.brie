define main [
  while true [
    2
  ]
  3
]
