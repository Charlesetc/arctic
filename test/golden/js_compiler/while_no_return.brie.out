const global_functions = {1: {}};
function main() {
	while (true) {2;};
	return 3;
}
console.log(main());
var test/golden/js_compiler/while_no_return.brie_main = new_closure(1)